File: leetcode3/3978. Unique Middle Element.py
class Solution:
    def isMiddleElementUnique(self, nums: list[int]) -> bool:
        ret = 0
        gijun = len(nums)//2
        gijun = nums[gijun]

        for n in nums:
            if n == gijun:
                ret+=1

        if ret == 1:
            return True
        else:
            return False

File: leetcode3/황은지/3978. Unique Middle Element.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var isMiddleElementUnique = function(nums) {
    const middleIndex=Math.floor(nums.length/2);
    const middleNum=nums[middleIndex];

    for(let i=0;i<nums.length;i++){
        if(i===middleIndex) continue;
        if(nums[i]===middleNum) return false;
    }
    return true;
};